commands/logs.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) OpenFaaS Author(s) 2019. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/openfaas/faas-cli/flags"
+	"github.com/openfaas/faas-provider/logs"
+
+	"github.com/openfaas/faas-cli/proxy"
+	"github.com/spf13/cobra"
+)
+
+var (
+	logFlagValues logFlags
+	nowFunc       = time.Now
+)
+
+type logFlags struct {
+	instance  string
+	since     time.Duration
+	sinceTime flags.TimestampFlag
+	follow    bool
+	tail      int
+}
+
+func init() {
+	initLogCmdFlags(functionLogsCmd)
+
+	faasCmd.AddCommand(functionLogsCmd)
+}
+
+var functionLogsCmd = &cobra.Command{
+	Use:     `logs <NAME> [--tls-no-verify] [--gateway]`,
+	Aliases: []string{"ls"},
+	Short:   "Tail logs from your functions",
+	Long:    "Tail logs from your functions",
+	Example: `faas-cli logs echo
+faas-cli logs echo --follow=false
```

Comment:
Didn't we have a way of limiting to N lines, or am I remembering wrong?

Reply:
Can you add an example for `--tail`?

Reply:
The `--tail` flag is the one used to limit the number of lines, it behaves the same as the `kubectl` flag

```bash
--tail=-1: Lines of recent log file to display. Defaults to -1 with no selector, showing all log lines
```

I will update the examples to show it being used.